go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/rs/xid v1.2.1
 	github.com/rs/zerolog v1.19.0
+	github.com/satori/go.uuid v1.2.0
```

Comment:
As we're already using `"github.com/rs/xid"` for generating GUID in mino, I would stick with it and not add a new dependency.